copilot_proxy/Dockerfile

```diff
@@ -5,8 +5,8 @@ FROM python:3.8-slim-buster
 WORKDIR /python-docker
 
 COPY requirements.txt requirements.txt
-RUN pip3 install -r requirements.txt
-RUN pip3 install --extra-index-url https://pypi.ngc.nvidia.com regex fire tritonclient[all]
+RUN --mount=type=cache,target=/root/.cache/pip/ pip3 install -r requirements.txt
```

Comment:
What for?

Reply:
Missed appending `regex fire tritonclient[all]` to requirements